website/docs/d/buckets.html.markdown

```diff
@@ -0,0 +1,37 @@
+---
+layout: "runscope"
+page_title: "Runscope: runscope_buckets"
+sidebar_current: "docs-runscope-datasource-buckets"
+description: |-
+  Get information about runscope buckets.
+---
+
+# runscope\_buckets
+
+Use this data source to get information about matching [buckets](https://www.runscope.com/docs/api/buckets)
+that you can with other runscope resources.
+
```

Comment:
type missing word `use`

 "that you can with other" assume it should be "that you can use with other"